express-hello-world/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res) {
+});
+
+app.listen(3000, () => {
+});
```

Comment:
Seems like you forgot to commit some of your changes, based on your screenshot.

Reply:
Did you want the console.logs in there?  Because the linter won't let me commit it with them there

Reply:
<img width="523" alt="Screen Shot 2022-05-06 at 3 21 19 PM" src="https://user-images.githubusercontent.com/20695764/167223436-1099b23e-0e0e-49c2-9a88-e9d40cd1328c.png">